manifests/repo/webtatic.pp

```diff
@@ -4,7 +4,7 @@
 #
 class yum::repo::webtatic {
   $osver = split($::operatingsystemrelease, '[.]')
-  $mirrorlist = $osver[0] ? {
+  $mirrorlist = (0+$osver[0]) ? {
     5 => 'http://repo.webtatic.com/yum/centos/5/$basearch/mirrorlist',
     6 => 'http://repo.webtatic.com/yum/el6/$basearch/mirrorlist',
     7 => 'http://repo.webtatic.com/yum/el7/$basearch/mirrorlist',
```

Comment:
I suppose you have this error with Puppet 4 or future parser enabled, in any case I'd rather quote the version numbers here:
  $mirrorlist = ($osver[0]) ? {
     '5' => 'http://repo.webtatic.com/yum/centos/5/$basearch/mirrorlist',
     '6' => 'http://repo.webtatic.com/yum/el6/$basearch/mirrorlist',
     '7' => 'http://repo.webtatic.com/yum/el7/$basearch/mirrorlist',